### ISSUE TITLE
Zobrazení PDF z K7

### ISSUE THREAD

**Issue by kerschfilip** [ceskaexpedice/ASJournals#178]
Vypadá to, že ASJ nerespektují plně nastavení, zda pracují z K7 nebo K5.

Mám časopis, který jsem nastavil na práci s naším test K7:
![image](https://github.com/user-attachments/assets/602af031-4ae4-4a96-980d-2302f5b16f1d)

Zatímco metadata se mi stáhla správně z K7 (kde jsou pro testovací účely schválně trochu jiná, než v produkční K5), tak PDF článku, který je v K7 navíc se mi v rozhraní nechce zobrazit: https://asjournals.inovatika.dev/ceskaliteraturaK7/article/uuid:6ed2135a-8844-4235-b5ec-8da5f200601d/pdf

Navíc to vypadá, že i citaci se ASJ snaží získat z URL Krameria, se kterým podle nastavení nemá vůbec pracovat:
![image](https://github.com/user-attachments/assets/23c6e863-501d-46ee-a843-73d80538194e)

**Comment by albertoh** [comment]
Opraveno.
Ale problem s citaci pretrvava. Ovavam se, ze ted to je zpusobeno tim, ze https://kramerius-test.lib.cas.cz neni volne dostupny

**Comment by kerschfilip** [comment]
@albertoh zkusil jsem udělat nový testovací časopis na K7 a dostat ho do ASJ, ale nějak se mi to vůbec nenačetlo - podívej se na to, prosím. Nefunguje náhled časopisu ani to neindexuje články:

- https://kramerius-test.lib.cas.cz/periodical/uuid:6e59d9eb-dbc9-4413-ac35-aa10045dfc92
- https://asjournals.inovatika.dev/protimluv/home

**Comment by kerschfilip** [comment]
Ještě jsem si všiml, že PDF se načítají, takže v podstatě blbne hlavně zobrazení v archivu: https://asjournals.inovatika.dev/protimluv/article/uuid:4237b050-0f79-4b81-a457-d7864aab396f/pdf

**Comment by albertoh** [comment]
Opraveno.
Zda se, ze od te doby kdyz jsem to udelal, K7 zmenil trochu API

**Comment by kerschfilip** [comment]
OK, je to lepší, ještě vypíšu problémy na které jsem narazil:

Na hlavní stránce se neukazuje náhled časopisu:
![image](https://github.com/user-attachments/assets/660ad7ee-9bd2-4fc9-a009-779f696290cc)

Při rozkliknutí aktuálního čísla se tam ukazuje genre.volume a genre.normal - ty asi patří vyšším úrovním a ne článkům?
![image](https://github.com/user-attachments/assets/1ada5d46-6b8b-45ff-851c-cd0d54dbaf22)

Na detailu článku ([zde](https://asjournals.inovatika.dev/protimluv/article/uuid:08d29c69-cc1d-403a-b485-f2bbc6695e2c/detail)) se neukazuje mezi metadaty autor a přidal jsem tam i seznam použité literatury pro kontrolu https://github.com/ceskaexpedice/ASJournals/issues/125, ale ten tam také nevidím.

**Comment by albertoh** [comment]
Obrazek: byla tam mezera navic pred https

**Comment by albertoh** [comment]
Seznam použité literatury:
V #125 mods xml mel tagy bez namespace mods. Napr:
  drive: 
```
<relatedItem id="ref0003" type="references">
      <name>
        <namePart type="family">Mocná</namePart>
        <namePart type="given">Dagmar</namePart>
      </name>
```

Ted:
```
<mods:relatedItem ID="ref0001" type="references">
      <mods:name>
        <mods:namePart type="family">Biehl</mods:namePart>
        <mods:namePart type="given">Janet</mods:namePart>
      </mods:name>

```

Chapu to tak, ze pred tim, bez prefixu, nei spravne, jo?

**Comment by kerschfilip** [comment]
> Autor: nechybi tam v mods:name atribut type=personal?

Ano chybí. Díky, opravím.

> V #125 mods xml mel tagy bez namespace mods

Máš pravdu, promiň. Vůbec mi ten prefix nedošel, ale teď jsem to připravoval v ProArcu a ten mě to XML bez prefixu mods nenechá uložit. Takže tam být musí. Díky

**Comment by kerschfilip** [comment]
Seznam použití literatury řešíme dál tady: https://github.com/ceskaexpedice/ASJournals/issues/125
V tomhle issue bych nechal problémy zobrazování dat z K7

V časopisu přebíraného z K7 mi tedy nyní nefunuguje:

- [ ] Při rozkliknutí aktuálního čísla se tam ukazuje genre.volume a genre.normal - ty asi patří vyšším úrovním a ne článkům?

![image](https://github.com/user-attachments/assets/bb7ac0f5-8f55-4ce9-aad9-294dccf4d6ca)

- [ ] v záložce Archiv se nezobrazují karty ročníků, ale články. Viz např. https://asjournals.inovatika.dev/protimluv/archiv nebo https://asjournals.inovatika.dev/ceskaliteraturaK7/archiv

**Comment by albertoh** [comment]
Opraveno

**Comment by kerschfilip** [comment]
Zobrazení už vypadá OK, ale zkoušel jsem si vytvořit a indexovat nový časopis z naší budoucí produkční K7 a nejde mi to. Není možné, že je to  teď nastavené tak, že to funguje jen s naší testovací instance? Z kramerius-test.lib.cas.cz mi indexovat časopisy jde, z kramerius-new.lib.cas.cz ne :/  Našel jsem tohle: https://github.com/ceskaexpedice/ASJournals/blob/0fcf372e8a914e6179f751f7795d2dd4e533794b/searchapp/src/main/resources/cz/incad/k5journals/searchapp/server_config.json#L5

**Comment by albertoh** [comment]
Ano, to se musi nastavit v konfiguraci. A v krameriovi se ma povolit IP adresu asjournals serveru
